commitlint/plugins.ts

```diff
@@ -272,6 +272,24 @@ export abstract class Plugins {
         ];
     }
 
+    public static properRevertMessage(headerStr: string, bodyStr: string) {
+        let offence = false;
+
+        if (headerStr.match(/^[Rr]evert ".+"$/) !== null) {
+            // does msg have a body?
+            if (bodyStr !== "") {
+                let lines = bodyStr.split("\n");
+                offence =
+                    lines[0].match(/^This reverts commit [^ ]{40}\.$/) !==
+                        null && lines.length == 1;
+            } else {
+                offence = true;
+            }
+        }
+
+        return [!offence, `Please explain why you're reverting`];
```

Comment:
@realmarv I've changed recently all error messages of all plugins, let's now be consistent with them here ^

Reply:
Done